.github/workflows/automated_release.yml

```diff
@@ -33,15 +33,13 @@ jobs:
         run: echo "RELEASE_VERSION=$([ ${GITHUB_REF##*/} = "develop" ] && printf '%s\n' $(cat .release-version)-develop || printf '%s\n' $(cat .release-version))" >> $GITHUB_ENV
 
       - name: Create release
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: softprops/action-gh-release@v1
```

Comment:
What's the difference between these two? (softprops/action-gh-release@v1 vs actions/create-release@v1)